common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -770,10 +804,139 @@ def test_discussion_on_other_team_page(self, is_staff):
+        """
+        Verifies that user can see all the information, present on detail page according to its membership status.
+
+        Arguments:
+            num_members (int): number of users in a team
+            is_member (bool) default True: True if user is member else False
+            max_size (int): number of users a team can have
+            invite_text (str): help text for invite link.
+        """
+        self.assertEqual(
+            self.team_page.team_details_info, {
+                'team_members_present': True if num_members else False,
+                'team_capacity': '{num_members} / {max_size} {members_text}'.format(
+                    num_members=num_members,
+                    max_size=max_size,
+                    members_text='Member' if num_members == max_size else 'Members'
+                ),
+                'team_location': 'Afghanistan',
+                'team_language': 'Afar'
+            }
+        )
+        self.assertEqual(self.team_page.team_members, num_members)
+
+        if is_member:
+            self.assertEqual(self.team_page.team_membership_text, 'You are a member of this team.')
+            self.assertTrue(self.team_page.team_leave_link_present)
+            self.assertTrue(self.team_page.team_invite_section_present)
+            self.assertEqual(self.team_page.team_invite_help_text, invite_text)
+        else:
+            self.assertEqual(self.team_page.team_membership_text, '')
+            self.assertFalse(self.team_page.team_leave_link_present)
+            self.assertFalse(self.team_page.team_invite_section_present)
+
+    def test_team_member_can_see_full_team_details(self):
+        """
+        Scenario: Team member can see full info for team.
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic of which I am a member
+        When I visit the Team page for that team
+        Then I should see a full team detail
+        And I should see the team members
+        And I should see the team membership text
+        And I should see the language & country
+        And I should see the Leave Team and Invite Team
+        """
+        self._team_config()
+        self.team_page.visit()
+
+        self.assert_team_details(
+            num_members=1,
+            invite_text='Send this link to friends so that they can join too.'
+        )
+
+    def test_other_users_can_see_limited_team_details(self):
+        """
+        Scenario: Users who are not member of this team can only see limited info for this team.
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic of which I am not a member
+        When I visit the Team page for that team
+        Then I should not see full team detail
+        And I should see the team members
+        And I should not see the team membership text
+        And I should not see the Leave Team and Invite Team
```

Comment:
"And I should not see my team membership text"
"And I should see the Leave Team and Invite Team links"